docs/io/votable/16091.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix `TableElement._write_binary` method to ensure proper handling of null values during BINARY2 serialization.
```

Comment:
No need to mention a hidden method. Though you might want to add one more sentence to clarify what was improper before the fix.
```suggestion
Ensure proper handling of null values during BINARY2 serialization.
```